src/main/java/com/example/deskworkoutservice/DeskworkoutRequest.java

```diff
@@ -1,16 +1,31 @@
 package com.example.deskworkoutservice;
 
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.Size;
+
 //デスクワークアウト情報の登録 更新に使用
 public class DeskworkoutRequest {
 
+    @NotBlank(message = "空白は許可されていません")
+    @Size(max = 20, message = "名前は20文字以内で入力してください")

```

Comment:
```suggestion
    @Size(max = 20, message = "名前は {max} 文字以内で入力してください")

```
こういう書き方ができるはず
そうするとmaxの値がかわってもそれにあわせてメッセージも変更されます。

Reply:
コメントありがとうございます！
修正しました。
ご確認よろしくお願いします。
54e270e4a2d3e3ffe76ec086c2de9aab6db881ce